apps/explorer/src/core/api/apollo/extensions/token-page.ext.ts

```diff
@@ -18,14 +19,38 @@ export class TokenPageExt_items implements TokenPage_items {
     this.decimals = proto.decimals
     this.name = proto.name
     this.symbol = proto.symbol
+    this.priceChange24h = proto.priceChange24h
   }
 
   get currentPriceBN(): BN | null {
     return this.currentPrice ? new BN(this.currentPrice) : null
   }
 
-  get balanceBN(): BN | null {
-    return this.balance ? new BN(this.balance) : null
+  get formattedBalance(): string {
```

Comment:
@olgakup this is where we can add BigNumber conversion and other calculations that are likely to be used in multiple places

Reply:
Sorry @olgakup, now I see this is exactly what you did already with the page class, so no need for me to explain!